### DISCUSSION TITLE
Is it possible to create flashcard-content inside a normal note?

### DISCUSSION THREAD

**Discussion by SiliasGH** [st3v3nmw/obsidian-spaced-repetition#724] [Q&A]
Hi,

During writing my notes, I sometimes feel the urge, "hey, I want to remember this particular fact"! :) Instead of creating a separate flashcard-note, exactly for that part of the note, I'd like to use the syntax of this plugin within the note... - I hope, you understand what I mean. I give you an example:

this is the content of the note:
line 1 blablabla
line 2 blablabla
line 3 
line 4 #flashcards
line 5 what is the sum of 1 and 2::3
line 6 
line 7 blablabla

Is it possible to query line 5 out of that note?

Thanks in advance,
Silias

**Comment by SiliasGH** [comment]
okay, got it, sorry for the question.
The answer is: Yes it is possible like I asked up there - I just got confused because of some highlights in my notes, which were interpreted as  clozes.

**Comment by ronzulu** [comment]
Hi @SiliasGH 

Seems like you have worked this out. Could you please mark the discussion as closed.

Cheers
Ronny
